src/modules/wmtheme.c

```diff
@@ -45,6 +45,17 @@ static void printWMThemeFromConfigFile(FFinstance* instance, const char* configF
         return;
     }
 
+    // Remove Plasma-generated prefixes
+    uint32_t idx = 0;
+
+    idx = ffStrbufFirstIndexS(&theme, "qml_");
+    if((idx != theme.length))
```

Comment:
Why the double brackets?

Reply:
Oops. Copy paste error. I was going for a one liner prior. Something like
```c
if((idx = ffStrbufFirstIndexS(&theme, "qml_") != theme.length))
```
Couldn't get it to work.

Reply:
Probably because the index methods remove the index of the terminating null byte if the search failed.